src/Files.App/Utils/FileTags/FileTagsDatabase.cs

```diff
@@ -210,28 +211,32 @@ public string Export()
 
 		private void IterateKeys(List<TaggedFile> list, string path, int depth)
```

Comment:
Isn't there any need to address this on the part of the caller? I'm afraid it will cause problems such as exporting empty settings if we simply ignore the exception.

Reply:
How would you suggest handling?

Reply:
Instead of `IterateKeys`, you can catch and ignore SecurityException in each of `GetAll` and `GetAllUnderPath`.
`Export` also calls `IterateKeys`, but the caller of `Export` catches the exception and ignores it, so it is better to throw the exception without catching it here.